changelogs/unreleased/early-make-bootstrap-leader.md

```diff
@@ -0,0 +1,4 @@
+## feature/replication
```

Comment:
Typo in the docbot request:
```
So, the following code is now possible:

```lua
box.cfg.make_bootstrap_leader()
```
^ `box.ctl.make_bootstrap_leader()`

Reply:
Thanks! Will fix with the next push.

Reply:
Done.